openstack/assets/dashboards/openstack-controller.json

```diff
@@ -0,0 +1 @@
d by hypervisor","height":14,"tile_def":{"viz":"timeseries","requests":[{"q":"avg:openstack.nova.current_workload{*} by {hypervisor}","type":"bars"}]},"width":46,"legend":false,"time":{"live_span":"1h"},"y":12,"x":69,"legend_size":"0","type":"timeseries","id":15},{"title_size":16,"title":true,"title_align":"left","title_text":"VCPUs used vs available by hypervisor","height":14,"tile_def":{"viz":"timeseries","requests":[{"q":"sum:openstack.nova.vcpus{$aggregate} by {host}","style":{"palette":"warm"},"type":"line"},{"q":"sum:openstack.nova.vcpus_used{$aggregate} by {hypervisor}","type":"area"}]},"width":46,"legend":false,"time":{"live_span":"1h"},"y":48,"x":69,"legend_size":"0","type":"timeseries","id":16},{"font_size":"36","bgcolor":"yellow","type":"note","tick_edge":"bottom","text_align":"center","height":7,"width":38,"html":"[Nova Server](https://www.datadoghq.com/blog/openstack-monitoring-nova/#nova-server-metrics)","y":77,"x":121,"tick":true,"tick_pos":"50%","id":17},{"title_size":16,"title":true,"title_align":"left","title_text":"HDD read rate by instance for $project","height":14,"tile_def":{"viz":"timeseries","requests":[{"q":"avg:openstack.nova.server.hdd_read_req{$project} by {host}","style":{"width":"normal","palette":"dog_classic","type":"solid"},"type":"area"}],"yaxis":{"max":"auto","scale":"linear","includeZero":true,"label":"","min":"auto"}},"width":46,"legend":false,"time":{"live_span":"1h"},"y":87,"x":117,"legend_size":"0","type":"timeseries","id":18},{"sizing":"fit","url":"https://app.datadoghq.com/static/images/saas_logos/bot/openstack.png","height":10,"width":19,"y":0,"x":0,"type":"image","id":19},{"title_size":16,"title":true,"title_align":"left","title_text":"RabbitMQ queue memory","height":14,"tile_def":{"viz":"timeseries","requests":[{"q":"sum:rabbitmq.queue.memory{*} by {rabbitmq_queue}","style":{"palette":"cool"},"type":"area"}]},"width":46,"legend":false,"time":{"live_span":"1h"},"y":12,"x":117,"legend_size":"0","type":"timeseries","id":20},{"font_size":"36","bgcolor":"yellow","type":"note","tick_edge":"bottom","text_align":"center","height":8,"width":40,"html":"\n\n\n\n[RabbitMQ](https://www.datadoghq.com/blog/openstack-monitoring-nova/#rabbitmq-metrics)","y":2,"x":120,"tick":true,"tick_pos":"50%","id":21},{"title_size":16,"title":true,"title_align":"left","title_text":"Consumers by queue","height":16,"tile_def":{"viz":"toplist","requests":[{"q":"top(sum:rabbitmq.queue.consumers{*} by {rabbitmq_queue}, 50, 'mean', 'asc')","style":{"palette":"dog_classic"}}]},"width":46,"time":{"live_span":"1h"},"y":29,"x":117,"type":"toplist","id":22},{"title_size":16,"title":true,"title_align":"left","title_text":"RabbitMQ consumer utilization","height":14,"tile_def":{"viz":"timeseries","requests":[{"q":"sum:rabbitmq.queue.consumer_utilisation{*} by {rabbitmq_queue}","style":{"palette":"cool"},"type":"area"}]},"width":46,"legend":false,"time":{"live_span":"1h"},"y":48,"x":117,"legend_size":"0","type":"timeseries","id":23}],"id":1065602}
```

Comment:
Small description update, remove greater: `Used for OpenStack deployments v13 and higher`

Reply:
Besides that 👍 from docs